### ISSUE TITLE
Better MySQL Index Name support

### ISSUE THREAD

**Issue by guitarrapc** [k0kubun/sqldef#51]
## Description

When MySQL Index Name contains `.`, `ALTER TABLE xxxx ADD index index.name(column)` or similar  statement will fail with `Error 1103: Incorrect table name 'index'`.

Let's say when Index Name is "Foo.Bar" then following mysqldef generate statement will fail.

```sql
ALTER TABLE xxxx ADD index Foo.Bar(column)
```

Do you support this scenario, or it should avoid via user?

## Why

Because MySQL requires quote index name with \` when using `.` for name.

```sql
ALTER TABLE xxxx ADD index `Foo.Bar`(column)
```

**Comment by k0kubun** [comment]
Thank you for reporting this. Could you try v0.5.11 which is being released at https://travis-ci.org/k0kubun/sqldef/builds/637809423?

**Comment by guitarrapc** [comment]
thank you for prompt fix! I've confirmed v0.5.11 fixed issue.
